projects.go

```diff
@@ -108,6 +109,20 @@ type ProjectNamespace struct {
 	UpdatedAt   *time.Time `json:"updated_at"`
 }
 
+// StorageStatistics represents a statistics record for a group or project.
+type StorageStatistics struct {
+	StorageSize        uint64 `json:"storage_size"`
+	RepositorySize     uint64 `json:"repository_size"`
+	LfsObjectsSize     uint64 `json:"lfs_objects_size"`
+	BuildArtifactsSize uint64 `json:"build_artifacts_size"`
```

Comment:
You are right that this will never be a negative number, but a signed `int64` is more then big enough (it can represent sizes up to 8388608 Terabyte)